core/services/directrequest/delegate.go

```diff
@@ -132,8 +132,8 @@ func (l *listener) Start() error {
 			Contract: l.oracle.Address(),
 			ParseLog: l.oracle.ParseLog,
 			LogsWithTopics: map[common.Hash][][]log.Topic{
-				oracle_wrapper.OracleOracleRequest{}.Topic():       {{log.Topic(l.onChainJobSpecID)}},
-				oracle_wrapper.OracleCancelOracleRequest{}.Topic(): {{log.Topic(l.onChainJobSpecID)}},
+				oracle_wrapper.OracleOracleRequest{}.Topic():       {{log.Topic(l.job.ExternalIDEncodeBytesToTopic()), log.Topic(l.job.ExternalIDEncodeStringToTopic())}},
+				oracle_wrapper.OracleCancelOracleRequest{}.Topic(): {{log.Topic(l.job.ExternalIDEncodeBytesToTopic()), log.Topic(l.job.ExternalIDEncodeStringToTopic())}},
```

Comment:
Does [JobIDTopicFilter](https://github.com/smartcontractkit/chainlink/blob/develop/core/store/models/log_events.go#L127) exist in JPV2, and if so does it get added to this list somehow?

Reply:
They are the same thing - IDToTopic(jsID) == ExternalIDEncodeBytesToTopic() and ExternalIDEncodeStringToTopic() == IDToHexTopic(jsID)